meta/meta.go

```diff
@@ -69,13 +71,20 @@ var (
 	mRandomIDPrefix   = "TARID"
 	mBootstrapKey     = []byte("BootstrapKey")
 	mSchemaDiffPrefix = "Diff"
+	mPolicies         = []byte("Policies")
+	mPolicyPrefix     = "Policy"
+	mPolicyGlobalID   = []byte("PolicyGlobalID")
```

Comment:
Maybe we can reuse `mNextGlobalIDKey` instead of creating a new one?

Reply:
It is kinda standalone, I will vote for new one.

Reply:
I think it has no different with table or database and its name is `mNextGlobalIDKey`.

But It's not a big deal. Just looks better for me

Reply:
Agreed... I didn't see it's more special than all the other objects like table / database / view / sequence / schema version...

Reply:
It is kinda standalone, I will vote for new one.